### DISCUSSION TITLE
Ultralytics Live Session 11: YOLOv8 Turns One! 🎉 A Year Filled with Innovation & Impact

### DISCUSSION THREAD

**Discussion by pderrenger** [ultralytics/ultralytics#7478] [Show and tell]
Join us on January 11th, 2024, at 09:00 PDT/18:00 CET for a groundbreaking Ultralytics Live Session! Celebrate the Ultralytics YOLOv8 anniversary with Founder & CEO @glenn-jocher and key team members @AyushExel, @fcakyon, @kalenmike, @Laughing-q and @RizwanMunawar.

Discover the latest developments in AI with the introduction of Ultralytics Explorer and its API, explore advancements in image classification, get an in-depth look at Oriented Bounding Boxes, recap the year's achievements in Ultralytics HUB, and learn about our exciting plans for 2024. Engage with our community in a session filled with expert insights and forward-thinking discussions. 

Don't miss this opportunity to be at the forefront of AI innovation! 

Mark your calendars

📆 January 11th
⏰ 09:00 PDT/ 18:00 CET
🔗 https://www.youtube.com/watch?v=zHg7izeQJDw

[![YouTube thumbnail 1920x1080](https://github.com/ultralytics/ultralytics/assets/107626595/0220b30c-06d7-4c22-81f2-0bc0ef7d7b2e)](https://www.youtube.com/watch?v=zHg7izeQJDw)

**Comment by RizwanMunawar** [comment]
I am looking forward to the Ultralytics Live Session, we will discuss last year achievements and future plans :)

**Comment by glenn-jocher** [comment]
Everyone come with questions, we'll have a long Q&A session during the talk!!
